Apromore-Clients/portal-model/src/main/java/org/apromore/portal/helper/PermissionCatalog.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2021 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.apromore.portal.helper;
+
+public class PermissionCatalog {
```

Comment:
If you're going to have catalog of permissions, perhaps it would be better to convert `PermissionType` into an enumeration and do it there rather than creating this class.  That'd also tighten up the typing on the various method signatures.

Reply:
Will create a seperate PR to remove PermissionCatalog and refactor PermissionType